docs/ref/contrib/admin/index.txt

```diff
@@ -3192,24 +3192,34 @@ your URLconf. Specifically, add these four patterns::
 
     from django.contrib.auth import views as auth_views
 
+    admin.site.site_header = "My Admin Site"
```

Comment:
There is no need to set `admin.site.site_header` in this snippet, it's not required for a password reset feature. It's enough to pass it in `extra_context`.